packages/geolocation/jest.config.js

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
@CETHER why do we have all this dead code(commented code) here? we shouldn't have dead code in our project.

Reply:
Unnecesary commented code removed. Thank you for the suggestion!